src/ui/Pricing/Comparison/Table.svelte

```diff
@@ -1,15 +1,38 @@
 <script>
+  import Svg from '@/ui/Svg/svelte'
   import Feature from './Feature.svelte'
   import { COMPARE_TABLE } from './comapre'
 
-  export let plans
+  export let plans = []
+  export let isShowingMore = false
+
+  function filterTableCompareItems(items, shouldFilter) {
+    if (shouldFilter) {
+      return COMPARE_TABLE.slice(0, 7)
+    }
+
+    return COMPARE_TABLE
+  }
+
+  $: tableItems = filterTableCompareItems(plans, isShowingMore)
```

Comment:
Reactive statements should come before the function declarations
```suggestion
  $: tableItems = filterTableCompareItems(plans, isShowingMore)
  
  function filterTableCompareItems(items, shouldFilter) {
    if (shouldFilter) {
      return COMPARE_TABLE.slice(0, 7)
    }

    return COMPARE_TABLE
  }
```